src/ros_filter.cpp

```diff
@@ -646,9 +668,12 @@ namespace RobotLocalization
     FilterUtilities::appendPrefix(tfPrefix, baseLinkFrameId_);
     FilterUtilities::appendPrefix(tfPrefix, worldFrameId_);
 
-    // Whether we're publshing the world_frame->base_link_frame transform
+    // Whether we're publishing the world_frame->base_link_frame transform
```

Comment:
I realize this is a typo, but I'd rather have a PR that addresses comments and such specifically.